src/__tests__/questions.test.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import Questions from '../Questions';
+
+describe('<Questions />', () => {
+  let output;
+  const listOfQuestions = [{
```

Comment:
Why do you define hard coded stub data here? You have the same data in the Questins.json. Why don't you just import it and use here?
DRY - don't repeat yourself.